src/Model/Editor.re

```diff
@@ -45,6 +54,18 @@ type scrollbarMetrics = {
 /*   pixelHeight: int, */
 /* }; */
 
+let getCursors = (v: t) => {
```

Comment:
Refactor to use Vim cursors as part of the state (just store the array of cursors)